light-curve/light_curve/light_curve_py/features/rainbow/bolometric.py

```diff
@@ -186,7 +195,152 @@ def peak_time(t0, amplitude, rise_time, fall_time):
         return t0 + np.log(fall_time / rise_time) * rise_time * fall_time / (rise_time + fall_time)
 
 
+@dataclass()
+class LinexpBolometricTerm(BaseBolometricTerm):
+    """Linexp function, symmetric form. Generated using a prototype version of Multi-view
+    Symbolic Regression (Russeil et al. 2024, https://arxiv.org/abs/2402.04298) on
+    a SLSN ZTF light curve (https://ztf.snad.space/dr17/view/821207100004043)"""
+
+    @staticmethod
+    def parameter_names():
+        return ["reference_time", "amplitude", "rise_time"]
+
+    @staticmethod
+    def parameter_scalings():
+        return ["time", "flux", "timescale"]
+
+    @staticmethod
+    def value(t, t0, amplitude, rise_time):
+        dt = t0 - t
```

Comment:
So SN-like case (fast rise, long decay) will require negative values of rise_time, yes?.. 
Do we really need both signs of rise_time for physically motivated function? I suspect it will behave badly if the fit will wander close zero - so maybe we just restrict the function to just one sign of it?

Reply:
That's a valid point, it is really a design choice I think. The same goes for sigmoid, for which we could accept negative rising time, which would allow to fit decaying light curves. Or we could have separate functions, rising sigmoid and decaying sigmoid. For me it is fine like this as a first try, if we really see that it is unstable we could constrain it more

Reply:
Yes, I agree. Ok, let's keep it like that, until some real problem is encountered with it (I do not foresee any actual usage for this functional form due to its diverging behaviour before peak anyway)